README.md

```diff
@@ -38,33 +38,153 @@ The published library is available on
 
 # Quickstart
 
 After you've installed the package, you'll need an Auth Token. Retrieve one on the
 [Auth Token page of your ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken)
 
+Here is a minimal code block [using the 'connect' convenience function](https://github.com/ngrok/ngrok-python/blob/main/examples/ngrok-connect-minimal.py), with `authtoken_from_env=True` to use an Auth Token from the `NGROK_AUTHTOKEN` environment variable, and making a connection to `localhost` port `9000`:
+
+```python
+tunnel = ngrok.connect(9000, authtoken_from_env=True)
+print (f"Ingress established at {tunnel.url()}")
+```
+
 There are multiple examples in [the /examples directory](https://github.com/ngrok/ngrok-python/tree/main/examples).
-A minimal use-case looks like the following:
+
+## Authorization
+
+To use most features of ngrok, you need to obtain an Auth Token. You can get an Auth Token by signing up for free at [ngrok.com](https://dashboard.ngrok.com/signup) and then retrieving it from the [Auth Token page of your ngrok dashboard](https://dashboard.ngrok.com/get-started/your-authtoken). Once you have an Auth Token, you can reference it in several ways.
+
+You can set the authtoken in the environment variable `NGROK_AUTHTOKEN` and then pass `authtoken_from_env=True` to the [connect](https://ngrok.github.io/ngrok-python/module.html) function:
 
 ```python
-async def create_tunnel():
-    session = await ngrok.NgrokSessionBuilder().authtoken_from_env().connect()
-    tunnel = await session.http_endpoint().listen()
-    print (f"Ingress established at {tunnel.url()}")
-    tunnel.forward_tcp("localhost:9000")
+ngrok.connect(authtoken_from_env=True, ...)
 ```
 
-Or with [the 'connect' convenience function](https://github.com/ngrok/ngrok-python/blob/main/examples/ngrok-connect-minimal.py):
+You can also set the default Auth Token to use for all connections by calling the [set_auth_token](https://ngrok.github.io/ngrok-python/module.html) function:
 
 ```python
-tunnel = ngrok.connect(9000, authtoken_from_env=True)
-print (f"Ingress established at {tunnel.url()}")
+ngrok.set_auth_token(token)
+```
+
+Or the Auth Token can be passed directly to the [connect](https://ngrok.github.io/ngrok-python/module.html) function:
+
+```python
+ngrok.connect(authtoken=token, ...)
+```
+
+## Connection
+
+The [connect](https://ngrok.github.io/ngrok-python/module.html) function is the most common way to use this library. It will start an ngrok session if it is not already running, and then establish a tunnel to the specified address. If there is an asynchronous runtime running the [connect](https://ngrok.github.io/ngrok-python/module.html) function returns a promise that resolves to the public tunnel object.
+
+With just an integer, the `connect` function will forward to `localhost` on the specified port, or the host can be specified as a string:
```

Comment:
With no arguments it defaults to `localhost` port `80`, mimicking the community library behavior. That is mentioned in the old `Installation` section, but since that is being removed I'll add an example line here.